vowpalwabbit/cats_tree.cc

```diff
@@ -340,16 +340,16 @@ base_learner* setup(setup_base_i& stack_builder)
   options_i& options = *stack_builder.get_options();
   VW::workspace& all = *stack_builder.get_all_pointer();
 
-  option_group_definition new_options("CATS Tree Options");
+  option_group_definition new_options("CATS Tree");
   uint32_t num_actions;  // = K = 2^D
   uint32_t bandwidth;    // = 2^h#
   std::string link;
   new_options.add(make_option("cats_tree", num_actions).keep().necessary().help("CATS Tree with <k> labels"))
       .add(make_option("tree_bandwidth", bandwidth)
                .default_value(0)
                .keep()
-               .help("tree bandwidth for continuous actions in terms of #actions"))
-      .add(make_option("link", link).keep().help("Specify the link function: identity, logistic, glf1 or poisson"));
+               .help("Tree bandwidth for continuous actions in terms of #actions"))
```

Comment:
The check for glfw below is redundant now and cant be reached. Perhaps adding the explanation to the help string here would be useful and then removing the check below?